api/Terminal.hpp

```diff
@@ -152,6 +152,7 @@ class Terminal {
 
     // The following fields are available in API version 10.1 and later.
     bool frozen = false; // Whether the terminal should stop rendering
+    bool forcedUpdate = false; // Ignores `frozen` to render the terminal once 
```

Comment:
This must be placed at the end of the class - the current position would break ABI.